optuna/study.py

```diff
@@ -51,7 +54,7 @@ def best_params(self):
     @property
     def best_value(self):
```

Comment:
I found that the 65th line (`raise ValueError('No trials are completed yet.')`) was not covered by the unit tests.
Could you add a test case for it, please?

Reply:
[NOTE]
We decided to not add a test case for this exception in this PR.
It's not so easy because the public APIs of Optuna doesn't allow creating a trial that has `COMPLETE` state and `None` value at the same time.